health/REFERENCE.md

```diff
@@ -61,6 +61,8 @@ Netdata parses the following lines. Beneath the table is an in-depth explanation
 | [`os`](#alarm-line-os)                              | no              | Which operating systems to run this chart.                                            |
 | [`hosts`](#alarm-line-hosts)                        | no              | Which hostnames will run this alarm.                                                  |
 | [`families`](#alarm-line-families)                  | no              | Restrict a template to only certain families.                                         |
+| [`module`](#alarm-line-module)                      | no              | Restrict alarm to only certain module.                                                |
+| [`plugin`](#alarm-line-plugin)                      | no              | Restrict alarm to only certain plugin.                                                |
```

Comment:
i suggest to change to:
- plugin
- module
- families

order

Reply:
I agree with the suggested order, it will make more sense.